itemManagement/search_indexes.py

```diff
@@ -9,6 +9,8 @@
 class ItemIndex(indexes.SearchIndex, indexes.Indexable):
     text = indexes.CharField(document=True, use_template=True)
     deleted = indexes.BooleanField(model_attr='deleted')
+    lastUsed = indexes.DateTimeField(model_attr="lastUsed")
+    title = indexes.CharField(model_attr="title")
```

Comment:
If you're going to make changes to something, please make sure you understand the underlying system (resources in comments of this page). 
1. text already includes title and tag information. See item_text.txt
2. Since we're not ordering the queryset by lastUsed (see comments below), lastUsed doesn't need to be here

Reply:
Still remove this @LiamWalsh98